content/en/apps/guides/hosting/4.x/self-hosting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Self Hosting in CHT 4.x"
+linkTitle: "Self Hosting"
+weight: 10
+description: >
+  Details for hosting the CHT on self run infrastructure
+---
+
+# Recommendations and considerations
+
+## Multi vs Single node couchdb requirements
+
+For smaller deployments a [single node CouchDB][single-couch] instance can be used, for larger deployments a [multi-node CouchDB][multi-couch] cluster is generally recommended
+
+| Consideration                                                                   | [Single node CouchDB][single-couch] | [Multi-node clustered CouchDB][multi-couch] |
+| ------------------------------------------------------------------------------- | ----------------------------------- | ------------------------------------------- |
+| Less than 2000 users                                                            | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 2000 users                                                            | {{< icon/no >}}                     | {{< icon/yes >}}                            |
+| Less than 1 million documents                                                   | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 1 million documents                                                   | {{< icon/no >}}                     | {{< icon/yes >}}                            |
```

Comment:
@garethbowen do you know if these numbers are accurate?

Reply:
I think it would be more important to think about docs per day not just outright doc count. If a db has a lot of docs but it's not growing much then single node is probably fine. I think the tipping point would be somewhere around 10k docs in a day. This accounts for the burst traffic point below...

The user count is probably a little low - we have single nodes working with ~4k users right now. Let's set it to 4k.